lisa/sut_orchestrator/libvirt/ch_platform.py

```diff
@@ -101,7 +102,21 @@ def _create_node_domain_xml(
         node_context = get_node_context(node)
 
         domain = ET.Element("domain")
-        domain.attrib["type"] = "ch"
+
+        libvirt_version = self._get_libvirt_version()
+        if parse_version(libvirt_version) > parse_version("10.0.2"):
```

Comment:
`parse_version(libvirt_version) > "10.0.2"` is enough.

Reply:
done